include/psa/crypto_extra.h

```diff
@@ -1920,7 +1920,8 @@ struct psa_pake_operation_s
 #if defined(MBEDTLS_PSA_BUILTIN_PAKE)
     unsigned int MBEDTLS_PRIVATE(input_step);
     unsigned int MBEDTLS_PRIVATE(output_step);
-    mbedtls_svc_key_id_t MBEDTLS_PRIVATE(password);
+    uint8_t* MBEDTLS_PRIVATE(password_data);
+    size_t MBEDTLS_PRIVATE(password_bytes);
```

Comment:
I think `password` + `password_len` would be more in line with our usual naming conventions.